src/components/molecules/SectionRow.tsx

```diff
@@ -9,18 +10,18 @@ export type SectionRowProps = {
 
 function SectionRow({ title, description }: SectionRowProps) {
   return (
-    <div css={rowStyle}>
-      <div css={titleWrapperStyle}>
-        <Text as="h6">{title}</Text>
-      </div>
+    <article css={rowStyle}>
+      <Text as="header" css={titleWrapperStyle}>
+        {title}
+      </Text>
```

Comment:
```suggestion
      <header css={titleWrapperStyle}>
        <Text as="h5">{title}</Text>
      </header>
```
header는 영역을 의미하는 태그라서 요렇게 안 쪽에 다시 헤딩을 넣어줘도 좋을 것 같아유!

Reply:
오홍! 그게 좋겠네여! 단순히 궁금한게 있는데 header는 페이지 색인에 도움을 준다고 하던데 만약에 header에 많은 정보가 있어도 큰 상관이 업없을까요?

Reply:
글쎄유 자세한 SEO 알고리즘은 구글신만이 아시는 거라....ㅜㅜ 사실 구글의 가이드라인은 "명확한 정보를 표현해달라" 하나이기는 합니다.
그래서 header에 정보가 많은 것이 좋은지, 어느 정도의 양부터 많다고 판단하는 것인지와 같은 정보는 알 수 가 없네요 ㅜㅜ